board/templates/base.html

```diff
@@ -72,17 +73,18 @@
 			  </div>
 			</nav>
 		{# The Document Body #}
-		<div class="row top-buffer">
-		    <div class="col-md-12">
-		    </div>
-		</div>
-		<div id="site-wrapper">
-			{% if user.is_authenticated %}
-			  {% block content %}
+		{% if user.is_authenticated %}
+			<div id="site-wrapper">
+				<div class="row top-buffer">
+				    <div class="col-md-12">
+				    </div>
+				</div>
+			</div>
+			{% block content %}
 				This is base html, if you see this, something is wrong!
 			{% endblock content %}
 			{% else %}
-				<div class="container-fluid">
+				<!-- <div class="container-fluid">
```

Comment:
Could this be the reason the top space collapses in the new UI? Accepting for now but opening an issue that i'll fix.